File: PrevozMVC/ePrevoz.WEB/Models/PrevozVM.cs
using ePrevoz.DATA.Model;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ePrevoz.WEB.Models
{
    public class PrevozVM
    {
        

        public class Info {
            public int Id { get; set; }

            public bool IsDeleted { get; set; }

        
           
      
            public float Cijena { get; set; }
           
            public string Opis { get; set; }

            public int BrojMjesta { get; set; }
            public bool SlobodnoMjesto { get; set; }
            public bool Aktivno { get; set; }

            public KorisnikVM Korisnik { get; set; }
            public int PrevoznoSredstvoId { get; set; }
            public String PrevoznoSredstvo { get; set; }

            public String Telefon { get; set; }

            public int TipPrevoza { get; set; }
            public int BrojZauzetihMjesta { get; set; }
     
            public DateTime DatumKretanja { get; set; }
            public DateTime DatumKreiranja { get; set; }

            public List<StanicaVM> Stanice { get; set; }

        }

        public List<Info> Prevozi { get; set; }
    }
}

File: PrevozApp/Prevoz/app/src/main/java/com/basic/prevoz/Models/PrevozVM.java
package com.basic.prevoz.Models;

import java.io.Serializable;

import java.util.Date;
import java.util.List;

/**
 * Created by Developer on 29.04.2017..
 */

public class PrevozVM implements Serializable {

    public int Id ;
    public String Opis ;
    public float Cijena ;
    public int BrojMjesta ;
    public boolean SlobodnoMjesto ;
    public boolean Aktivno ;


    public Date DatumKretanja ;
    public Date DatumKreiranja ;


    public int TipPrevoza ;

    public KorisniciVM Korisnik ;
    public int KorisnikId ;
    public List<StanicaVM> Stanice ;
    public int PrevoznoSredstvoId ;

    public String Telefon;
}
